src/graphics/aurora/guiquad.cpp

```diff
@@ -199,6 +214,18 @@ void GUIQuad::render(RenderPass pass) {
 		glLogicOp(GL_XOR);
 	}
 
+	if (_scissor) {
+		glEnable(GL_SCISSOR_TEST);
+		GLint viewport[4];
+		glGetIntegerv(GL_VIEWPORT, viewport);
+		glScissor(
+				viewport[2]/2 + _x1 + _scissorx,
+				viewport[3]/2 + _y1 + _scissory,
+				_scissorwidth,
+				_scissorheight
+		);
```

Comment:
Please use tabs for indenting, spaces for alignment. And put the first argument(s) on the same line as the function call itself.

I.e. it should rather looks like this:

```
		glScissor(viewport[2]/2 + _x1 + _scissorx,
		          viewport[3]/2 + _y1 + _scissory,
		          _scissorwidth,
		          _scissorheight);

```
Note that the tabs go to the level of glScissors, and then there's spaces to align the arguments to the opening parenthesis.